modules/plot/main/coffee/charts/PieChart.coffee

```diff
@@ -79,7 +81,7 @@ class PieChart extends hx.EventEmitter
 
   render: ->
     self = this
-    selection = hx.select(@selector)
+    selection = hx.select(@selector).class('hx-pie-chart')
```

Comment:
The class should be applied with `classed('hx-pie-chart', true)` to avoid removing existing classes